.gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml

```diff
@@ -53,6 +53,17 @@ spec:
             value: "{{ .Values.rabbitmq.username }}"
           - name: RabbitMQ__Password
             value: "{{ .Values.rabbitmq.password }}"
+          - name: TicketStorage__Type
```

Comment:
This defaults to object storage if not defined. So I prefer to skip it. The ObjectStorage parameters below are not conditional.

Reply:
Done, removed. Left to defaults